src/sqlancer/MainOptions.java

```diff
@@ -49,6 +49,12 @@
     @Parameter(names = "--password", description = "The password used to log into the DBMS")
     private String password = "sqlancer"; // NOPMD
 
+    @Parameter(names = "--host", description = "The host used to log into the DBMS")
+    private String host = null; // NOPMD
+
+    @Parameter(names = "--port", description = "The port used to log into the DBMS")
+    private int port = -1; // NOPMD
```

Comment:
Rather than checking for `-1`, could you please introduce a constant like `NO_PORT_SET` and use it also in the `createDatabase` methods?